src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const fullDaysPrice = days * costPerDay;
+
+  if (days < 3) {
+    return fullDaysPrice;
+  }
+
+  return days >= 7 ? fullDaysPrice - bigDiscount
+    : fullDaysPrice - smallDiscount;
```

Comment:
Remember about proper formatting of ternaries
```suggestion
  return days >= 7 
  	? fullDaysPrice - bigDiscount
    : fullDaysPrice - smallDiscount;
```